doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-execution-rest-api-proc.adoc

```diff
@@ -0,0 +1,516 @@
+            <rawValue class="string">15</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[4]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[4]/expressionIdentifier"/>
+            <rawValue class="string">&quot;speed&quot;</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[4]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[5]/expressionIdentifier"/>
+            <rawValue class="string">100</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[4]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[6]/expressionIdentifier"/>
+            <rawValue class="string">140</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[7]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[7]/expressionIdentifier"/>
+            <rawValue class="string">7</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[7]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[8]/expressionIdentifier"/>
+            <rawValue class="string">1000</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[9]/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping[9]/expressionIdentifier"/>
+            <rawValue class="string">&quot;Yes&quot;</rawValue>
+          </FactMappingValue>
+          <FactMappingValue>
+            <factIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping/factIdentifier"/>
+            <expressionIdentifier reference="../../../../../simulationDescriptor/factMappings/FactMapping/expressionIdentifier"/>
+            <rawValue class="string">5</rawValue>
+          </FactMappingValue>
+        </factMappingValues>
+        <simulationDescriptor reference="../../../simulationDescriptor"/>
+      </Scenario>
+    </scenarios>
+  </simulation>
+  <imports>
+    <imports/>
+  </imports>
+</ScenarioSimulationModel>
+----
```

Comment:
Can you please add an example of the result too?

Reply:
Okay, sure

Reply:
And for failing tests as well

Reply:
@dupliaka and @danielezonca,

HTTPCode 200 means Test Scenario successfully executed.

If the test failed then it produces any of the following response, right?
400
401
500

Do I have to add the response for all of these HTTPCodes?
Can you please confirm?

Reply:
Hey @dupliaka I confirmed with @danielezonca, we are following the same approach of DMN documentation. Failed cases are not mentioned.

Reply:
Sorry, once again. 
The point is not in the code the point is in the responce when one of the test scenario fails

Reply:
So the output of the service in case if one of the test scenario is failing would be:
'<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<response type="FAILURE" msg="Test Scenario execution failed">
      <scenario-simulation-result>
            <run-count>1</run-count>
            <ignore-count>0</ignore-count>
            <run-time>5</run-time>
            <failures>
                  <description>#1: Above speed limit: 10km/h and 30 km/h(org.drools.scenariosimulation.backend.runner.AbstractScenarioRunner)</description>
                  <error-message>#1: Scenario 'Above speed limit: 10km/h and 30 km/h' failed</error-message>
            </failures>
      </scenario-simulation-result>
</response>
'

Reply:
Ok, to keep the balance between overflowing users by info in docs and specifying it on a certain level to sync it with DMN documentation that is pretty much the same we need to skip "Failed cases" section.